snippets/shared/phases-explanation.mdx

```diff
@@ -0,0 +1 @@
+In Traffic Policy, a phase represents a distinct point in the lifecycle of a request as it moves through an ngrok endpoint. Phases allow you to inspect, process, and manage traffic at key moments.
```

Comment:
Why do we capitalize "Traffic Policy" but not "endpoint"? Something feels inconsistent here